llvm/tools/llvm-gsymutil/llvm-gsymutil.cpp

```diff
@@ -177,6 +179,16 @@ static void parseArgs(int argc, char **argv) {
 
   LookupAddressesFromStdin = Args.hasArg(OPT_addresses_from_stdin);
   StoreMergedFunctionInfo = Args.hasArg(OPT_merged_functions);
+
+  for (const llvm::opt::Arg *A : Args.filtered(OPT_callsites_from_yaml_EQ))
+    if (A->getValue() && A->getValue()[0] != '\0')
+      CallSiteYamlPaths.emplace_back(A->getValue());
+    else {
+      llvm::errs()
```

Comment:
It appears you follow the existing code pattern. Can we refactor this error handling code like `exitWithError` in `llvm-profdata`?

Reply:
There are a few places to refactor this. Can I do this in an follow up PR right after I land this ?